### DISCUSSION TITLE
AI翻唱被抨擊和抵制

### DISCUSSION THREAD

**Discussion by lyien1021** [1101-GS4719-C/Python-Community-GS4719-C#1244] [General]
https://hypebeast.com/zh/2023/4/drake-ai-rapping-ice-spice-munch-voice-response-info

最近的新聞報導說環球音樂正在要求Spotify, Apple Music等串流平台禁止AI對其公司之版權音樂的訪問權限，目的是為了阻止開發人員從受他們版權保護的歌曲中，抓取旋律或歌詞來訓練AI並生成音樂。

https://tw.news.yahoo.com/%E8%AE%93%E8%95%BE%E5%93%88%E5%A8%9C%E7%BF%BB%E5%94%B1%E7%A2%A7%E6%98%82%E7%B5%B2-%E7%92%B0%E7%90%83%E5%90%91-ai%E9%9F%B3%E6%A8%82-%E5%AE%A3%E6%88%B0-%E6%8F%AAspotify-070227222.html

隨著AI技術越來越發達，現在已經能讓AI從現有的歌曲中進行翻唱，彷彿真正邀請知名歌手或樂團來表演一樣，但根本未經對方允許。我覺得這其實是一個隱憂，如果能輕易盜用一個人的聲音，萬一用來犯罪要如何制裁？錄音證據還能採用嗎，會不會是AI造假的？

回到AI翻唱的議題上，雖然現在可能很多人覺得可以隨意指定某個歌手唱某首歌很新鮮有趣，但這樣當那位歌手真的翻唱的時候，是不是就失去了驚喜感，而且有的時候歌迷們喜歡的不就是現場表演的氣氛和舞台的感染力嗎，若都以AI來取代，未免也太無感，這樣跟聽音源有什麼差別？不知道大家對這個議題的看法是什麼？

**Comment by GuanYann** [comment]
AI翻唱的議題也引發了人們對音樂現場表演的真實性和情感感染力的思考。我認為，AI音樂生成技術是一個不斷發展的領域，需要對其進行適當的監管和管理，以確保其對音樂產業和聽眾的正面影響。

**Comment by curtis1017** [comment]
我覺得這個議題需要特別注意，尤其在我們資料量大的時代，除此之外還有假新聞跟假影片的產生，常常造成人們對資訊了解的錯誤，常常有假新聞，然後引起社會的恐慌，所以我覺得關於這個議題需要特別關注。

**Comment by CCello** [comment]
我認為AI翻唱涉及到音樂著作權問題，因為使用AI技術創作的音樂作品可能侵犯原作曲作者的著作權，尤其是當AI翻唱的曲目沒有經過原作曲作者的授權時，可能會引發版權爭議，且AI翻唱的音樂作品缺乏人類創作者的情感和創意，通常只能產生機械化的音樂效果，缺乏人性化的表現，因此可能無法引起聽眾的共鳴。
